src/parachain/fee.ts

```diff
@@ -1,33 +1,48 @@
-import { decodeFixedPointType } from "..";
 import { DefaultOracleAPI, OracleAPI } from "./oracle";
 import Big from "big.js";
 import { ApiPromise } from "@polkadot/api";
+import {
+    Bitcoin,
+    BTCAmount,
+    BTCUnit,
+    Currency,
+    ExchangeRate,
+    MonetaryAmount,
+    Polkadot,
+    PolkadotAmount,
```

Comment:
Probably requires a change in the monetary lib, but the `BTCAmount` and `DOTAmount` should have consistent naming. So rather `DOTAmount` than `PolkadotAmount`. Same for the units `PolkadotUnit` -> `DOTUnit`

Reply:
Added a ticket